pkg/providers/kube/handle.go

```diff
@@ -276,9 +278,31 @@ func (h *provider) Delete(ctx monitorContext.Context, wfCtx wfContext.Context, v
 		return err
 	}
 	deleteCtx := handleContext(ctx, cluster)
+
+	filterValue, err := v.LookupValue("filter")
+	if err != nil && !strings.Contains(err.Error(), "failed to lookup value: var(path=filter) not exist"){
```

Comment:
You can ignore the err handle here like:

```
if filterValue, err := v.LookupValue("filter"); err == nil {
filter := &filters{}
if err := filterValue.UnmarshalTo(filter); err != nil {
			return err
		}
		labelSelector, err := metav1.LabelSelectorAsSelector(&metav1.LabelSelector{MatchLabels: filter.MatchingLabels})
		if err != nil {
			return err
		}
		if err := h.cli.DeleteAllOf(deleteCtx , obj, &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: filter.Namespace, LabelSelector:  labelSelector}}); err != nil {
			return v.FillObject(err.Error(), "err")
		}
		return nil
}
```

Reply:
fixed